packages/google_sign_in/google_sign_in_android/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -521,16 +693,21 @@ private void onSignInResult(Task<GoogleSignInAccount> completedTask) {
     }
 
     private void onSignInAccount(GoogleSignInAccount account) {
-      Map<String, Object> response = new HashMap<>();
-      response.put("email", account.getEmail());
-      response.put("id", account.getId());
-      response.put("idToken", account.getIdToken());
-      response.put("serverAuthCode", account.getServerAuthCode());
-      response.put("displayName", account.getDisplayName());
+      final Messages.UserData.Builder builder =
+          new Messages.UserData.Builder()
+              // TODO(stuartmorgan): Test with games sign-in; according to docs these could be null
+              // as the games login request is currently constructed, but the public Dart API
+              // assumes they are non-null, so the sign-in query may need to change to
+              // include requestEmail() and requestProfile().
```

Comment:
This is very likely a latent bug (maybe nobody uses games sign in, or maybe it usually populates them?) but I didn't want to change any behavior, so I just documented it.